rabbitmq/check.py

```diff
@@ -337,6 +328,20 @@ def _get_metrics(self, data, object_type, custom_tags):
                     self.log.debug("Caught ValueError for %s %s = %s  with tags: %s" % (
                         METRIC_SUFFIX[object_type], attribute, value, tags))
 
+    def _get_queue_bindings_metrics(self, data, instance, object_type):
+        base_url, _, _, auth, ssl_verify, custom_tags = self._get_config(instance)
```

Comment:
`base_url` and `instance_proxy` should be passed as parameter to the function since we already collect them in get_stats.